packages/kit-forms/src/server.ts

```diff
@@ -0,0 +1,34 @@
+import { createComponentDefinition } from '@rise-tools/react'
+
+import type * as Kit from './form'
+
+export const RiseForm = createComponentDefinition<typeof Kit.RiseForm>(
+  '@rise-tools/kit-forms/RiseForm'
```

Comment:
I have mixed feelings about the long names caused by this namespace pattern. good for safety and bad for anybody who has to manually write component definitions

maybe we should get rid of the `@` because it doesn't really provide any value

Reply:
BTW in the future we can probably auto generate this file as it's quite repetitive.